Xamarin.Forms.Core/Device.cs

```diff
@@ -16,8 +16,9 @@ public static class Device
 		public const string UWP = "UWP";
 		public const string WinRT = "WinRT";
 		public const string macOS = "macOS";
+        public const string GTK = "GTK";
 
-		[EditorBrowsable(EditorBrowsableState.Never)]
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
seems all PR has problems with code guidelines,we use spaces not  tabs, we also don't use private